docs/development.rst

```diff
@@ -106,11 +106,7 @@ are making:
 
     $ git remote add upstream https://github.com/ga4gh/server.git
 
-All development is done against the ``develop`` branch.  The ``stable``
-branch is meant to be kept stable since it is the branch releases are
-based on -- don't touch it!  These are the two mainline branches.
-Our branching model is loosely based on the one described
-`here <http://nvie.com/posts/a-successful-git-branching-model/>`__.
+All development is done against the ``master`` branch.
```

Comment:
Yep, it seems like as good a time to rename the branches as any. What do you reckon?

Sphinx variable is a nice idea, but I don't have much intention of changing this again. Feel free to open a PR doing this if you like though!